src/main/java/liquibase/ext/spanner/sqlgenerator/CreateTableGeneratorSpanner.java

```diff
@@ -45,26 +45,30 @@ public ValidationErrors validate(
   public Sql[] generateSql(
       CreateTableStatement statement, Database database, SqlGeneratorChain sqlGeneratorChain) {
     Sql[] res = super.generateSql(statement, database, sqlGeneratorChain);
-    // Move the PRIMARY KEY statement from inside the table creation to outside.
-    StringBuilder buffer = new StringBuilder(", PRIMARY KEY (");
-    buffer.append(
-        database.escapeColumnNameList(
-            StringUtils.join(statement.getPrimaryKeyConstraint().getColumns(), ", ")));
-    buffer.append(")");
+    if (statement.getPrimaryKeyConstraint() != null && statement.getPrimaryKeyConstraint().getColumns() != null) {
```

Comment:
Pure style (and I'm not a Google style expert) question. Wouldn't this be preferred -->
// If no PK constraint, leave it as it is and let Spanner itself throw the error
if (statement.getPrimaryKeyConstraint() == null || statement.getPrimaryKeyConstraint().getColumns() == null) {
  return res;
}
<followed-by-current-logic>

Reply:
The (open source) [style guide](https://google.github.io/styleguide/javaguide.html) does not explicitly state so, but I agree that in this case it's certainly more readable.